ios/FluentUI/Vnext/Drawer/MSFSlideOutPanel+Modifiers.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+extension MSFSlideOutPanel {
```

Comment:
I think you can make this extension private. It'll save a bit on the binary size.